### ISSUE TITLE
Red numbers and BG points disappear in ambient mode

### ISSUE THREAD

**Issue by Vaughanabe13** [StephenBlackWasAlreadyTaken/NightWatch#64]
This is on a Smartwatch3. When the watch goes idle, the display dims and goes into the ambient mode, and when that happens, if blood sugar is currently under threshold, it will not be visible on the screen. It appears only the items colored in RED are disappearing in the ambient mode - so I see a partial trend line and the time, etc., but any data points that dip into the red zone are not drawn, and more importantly the current BG is not drawn. Not a big deal because you can just wake up the watch to view the current BG, but not being able to see it when glancing at the watch in ambient mode is problematic. I also have a Moto360 but I don't remember if this problem is specific to the SW3 or present on both watches so I'll double check the next time I dip low while using the 360.

**Comment by StephenBlackWasAlreadyTaken** [comment]
Just red in particular? I know the smartwatch3 has a different kind of display when its in ambient mode

**Comment by StephenBlackWasAlreadyTaken** [comment]
(do other colors work?)

**Comment by Vaughanabe13** [comment]
Yes, I can try to take a picture next time. I'm not sure how the SW3
ambient mode works, but when the backlight turns off, the red BG number
disappears completely. It's not an issue of color contrast, because I've
tried looking at different angles and in different lighting conditions and
the number is just plain gone when the backlight is off. If BG is in range
and displaying as a white number, then that white number is visible in
ambient mode. Same with the yellow number for high BG.

**Comment by Vaughanabe13** [comment]
Here are pictures (taken a few seconds apart to show backlight on vs. off). In the 'off' picture you can see the red threshold line, red BG number, and red data points all disappear.

![1014151845a](https://cloud.githubusercontent.com/assets/14842434/10516357/8284fa7c-731d-11e5-99b5-afcdd3a0119a.jpg)
![1014151845c](https://cloud.githubusercontent.com/assets/14842434/10516358/82851da4-731d-11e5-8fc2-e14cae78c28e.jpg)

**Comment by Vaughanabe13** [comment]
I found this post which talks about the ambient modes of android wear. It appears the SW3 uses a slightly different mode than other watches.

https://www.reddit.com/r/AndroidWear/comments/37wmb3/android_wear_has_3_different_ambient_modes_does/

I'm wondering if it would just be easiest to add a color selection or alt color mode for high/low BG numbers so I can select a color other than red?

**Comment by francesc0-cgm** [comment]
It happens due to its transflective display. It's not a bug of nightwatch but an issue of the display technology

**Comment by Vaughanabe13** [comment]
I understand that, but if the watch face has the ability to change color or
use an "alternate color" mode toggle, it's a nonissue and will be
compatible with all watches then. To me it's a safety issue because you
can't see your BG number when it is below threshold.

**Comment by francesc0-cgm** [comment]
You're right. I solved the issue setting low alarm level at 40...of course i use it for my 3 Years old and during the night i have to use Xdrip alarms ti wake up if it is low...never felt vibrations from any smartwatch i tried.

**Comment by britishguy4** [comment]
I also have this issue as a SW3 user. My simple workaround is to wake up the watch face but would be useful to have option to change low value colour on watch face

**Comment by francesc0-cgm** [comment]
Any other smartwatch with wear works without this issue. It is the transflective display...

**Comment by Docdavesi** [comment]
Smartwatch 3 face loses the color of the limit lines and the face turns white after going into an ambient mode

**Comment by Docdavesi** [comment]
![20160404_071307](https://cloud.githubusercontent.com/assets/11997976/14255054/40ac7e5c-fa47-11e5-9a57-939f30adc030.jpg)

**Comment by Docdavesi** [comment]
![20160404_071313](https://cloud.githubusercontent.com/assets/11997976/14255088/651e88a2-fa47-11e5-83af-988d81e38678.jpg)

**Comment by Docdavesi** [comment]
![20160404_071641](https://cloud.githubusercontent.com/assets/11997976/14255136/961ce232-fa47-11e5-8cf5-0993bd004b18.jpg)

**Comment by Docdavesi** [comment]
White face turns back to black once in normal blood sugar range, but the limit lines are still without color and the blood sugar below limit stays white.

**Comment by Docdavesi** [comment]
Correction found.  Open wear on watch via nightwatch on my phone (top right 3 dots).  On my SW3 unchecked all features for round faces and the limit lines and out of range line colors returned!
